ApiExtractor/abstractmetabuilder.cpp

```diff
@@ -34,20 +34,20 @@
 #include "parser/parser.h"
 #include "parser/tokens.h"
 
-#include <QDebug>
-#include <QFile>
-#include <QFileInfo>
-#include <QTextCodec>
-#include <QTextStream>
-#include <QVariant>
-#include <QTime>
-#include <QQueue>
-#include <QDir>
+#include <QtCore/QDebug>
```

Comment:
...that said, the only modules Shiboken should care about are ones that have not been refactored (i.e. Shiboken is not using headers that exist in different modules between Qt4 and Qt5), and this style of include *in theory* works with Qt4 also. (More to the point, I noticed there is at least one such include prior to these changes...)